samples/model-builder/conftest.py

```diff
@@ -822,7 +822,75 @@ def mock_get_params(mock_params, mock_experiment_run):
 @pytest.fixture
 def mock_get_time_series_metrics(mock_time_series_metrics, mock_experiment_run):
     with patch.object(
-        mock_experiment_run, "get_time_series_data_frame"
+            mock_experiment_run, "get_time_series_data_frame"
     ) as mock_get_time_series_metrics:
         mock_get_time_series_metrics.return_value = mock_time_series_metrics
         yield mock_get_time_series_metrics
+
+
+"""
+----------------------------------------------------------------------------
+Model Versioning Fixtures
+----------------------------------------------------------------------------
+"""
+
+
+@pytest.fixture
+def mock_model_registry():
+    mock = MagicMock(aiplatform.models.ModelRegistry)
+    yield mock
+
+
+@pytest.fixture
+def mock_version_info():
+    mock = MagicMock(aiplatform.models.VersionInfo)
+    yield mock
+
+
+@pytest.fixture
+def mock_init_model_registry(mock_model_registry):
+    with patch.object(aiplatform.models, "ModelRegistry") as mock:
+        mock.return_value = mock_model_registry
+        yield mock
+
+
+@pytest.fixture
+def mock_get_model(mock_model_registry):
+    with patch.object(mock_model_registry, "get_model") as mock_get_model:
+        mock_get_model.return_value = mock_model
+        yield mock_get_model
+
+
+@pytest.fixture
+def mock_get_model_version_info(mock_model_registry):
+    with patch.object(mock_model_registry, "get_version_info") as mock_get_model_version_info:
+        mock_get_model_version_info.return_value = mock_version_info
+        yield mock_get_model_version_info
+
+
+@pytest.fixture
+def mock_list_versions(mock_version_info):
+    with patch.object(aiplatform.models.ModelRegistry, "list_versions") as mock_list_versions:
```

Comment:
I'm wondering why the `mock_list_versions` uses `patch.object(aiplatform.models.ModelRegistry, "list_versions")` while all the other mocks use `patch.object(mock_model_registry, ...)`

Reply:
I updated the `mock_list_versions` accordingly. Thanks @nayaknishant